go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/prometheus/client_golang v0.9.2
```

Comment:
Just do not commit the go.mod and go.sum files, and the vendor test will pass.

Reply:
Thanks! I will do it in next PR)